test_programs/noir_test_success/regression_4080/src/main.nr

```diff
@@ -1,7 +1,8 @@
 // This test checks that `var^var` is assigned the correct type.
 // https://github.com/noir-lang/noir/issues/4080
 
-#[test(should_fail_with = "attempt to add with overflow")]
+// NOTE: expected error is the following, but the 'failed_assertion' is None in check_expected_failure_message
+#[test(should_fail_with = "Cannot solve opcode: FunctionInput value has too many bits")]
```

Comment:
I still would like `attempt to add with overflow` to be the error we receive here rather than an internal solver error. Were we not getting any failure output on this test?

Reply:
> Were we not getting any failure output on this test?

Yes, the ACVM error was blocking any other errors

Reply:
We should avoid this regression it is going to be very confusing for a user to see `Cannot solve opcode: FunctionInput value has too many bits`.

Reply:
Ok, I've reverted the test and disabled the ACVM bitsize checks for the associated opcodes: https://github.com/noir-lang/noir/issues/5985